server/app.js

```diff
@@ -21,4 +22,44 @@ app.get('/api/productView/products/:id', (req, res) => {
     })
 })
 
+app.delete('/api/productView/products/:id', (req, res) => {
+    console.log('delete request happening')
+    db.deleteMany({id: req.params.id})
+    .then((results) => {
+        res.status(200).send(results);
+    })
+    .catch((error) => {
+        console.log(error);
+        res.status(404).send();
+    })
+})
+
+app.put('/api/productView/products/:id', (req, res) => {
+    console.log('put request happening')
+    console.log('body', req.body)
+    db.updateMany({id: req.params.id}, {$set: req.body, $currentDate: {lastModified: true}})
+    .then((results) => {
+        res.status(200).send(results);
+    })
+    .catch((error) => {
+        console.log(error);
+        res.status(404).send();
+    })
+})
+
+app.post('/api/productView/products/:id', (req, res) => {
+    console.log('post request happening')
+    console.log('body', req.body)
```

Comment:
ultimately I think these console.logs are going to slow down your api. HOWEVER I am not sure.

Reply:
@dbooyah93 I agree, dude. They're definitely coming off.